apps/laboratory/tests/email.spec.ts

```diff
@@ -2,6 +2,11 @@ import { expect } from '@playwright/test'
 import { testMEmail } from './shared/fixtures/w3m-email-fixture'
 import { SECURE_WEBSITE_URL } from './shared/constants'
 
+const mailsacApiKey = process.env['MAILSAC_API_KEY']
+if (!mailsacApiKey) {
+  throw new Error('MAILSAC_API_KEY is not set')
+}
+
```

Comment:
I don't  think we need this

Reply:
you mean to remove the if condition? In that case, i should set a default value for the key

i need the mailsacApiKey